README.md

```diff
@@ -25,5 +25,10 @@ Usage instructions for deploying SUSE products from cloud service providers' mar
 [[Offer](https://aws.amazon.com/marketplace/pp?sku=doo3m38kfogikuu639ee8bwgk)]
 [[Usage Instructions](neuvector-prime/aws/suse-ltd/payg/usage-instructions.md)]
 
+##### Long term/private offers
+
+[[Offer](https://aws.amazon.com/marketplace/pp?sku=eqi0ds56aeicv916d6506hcjq)]
+[[Usage Instructions](neuvector-prime/aws/suse-ltd/lt/usage-instructions.md)]
```

Comment:
These are not visible should not have public information about his.

Reply:
You mean the offer link? The usage instructions will be public.

Reply:
b41d0a5

Reply:
Actually I meant everything. Why would we have a public document for use of something that cannot be used publicly?

Now I get the idea that the few that enter into a private offer contract also need documentation, that said since that is a 1 on 1 transaction shouldn't the documentation be part of he transaction and be passed along at that time?

Reply:
And where are we going to keep and maintain that documentation? A separate repository? Why?

Reply:
There's nothing secret or proprietary about the long term offers, or their usage instructions.